build-tools/src/main/kotlin/org/jetbrains/kotlin/ExecClang.kt

```diff
@@ -50,6 +50,17 @@ class ExecClang(private val project: Project) {
         }
     }
 
+    fun resolveToolchainExecutable(target: KonanTarget, executableOrNull: String?): String {
+        val executable = executableOrNull ?: "clang"
+
+        if (listOf("clang", "clang++").contains(executable)) {
+            val platform = platformManager.platform(target)
+            return "${platform.absoluteTargetToolchain}/usr/bin/$executable"
```

Comment:
(minor) May be support paths for non-Apple toolchains too, like it is done [here](https://github.com/JetBrains/kotlin-native/blob/d3c164de78d5eb8088168d809ec70934ec2ddbf0/backend.native/compiler/ir/backend.native/src/org/jetbrains/kotlin/backend/konan/BitcodeCompiler.kt#L37) for `BitcodeCompiler`? We could check `target.family` instead of `platform.configurables is AppleConfigurables`.

Not sure if all our toolchains include clang though.

Reply:
Fixed, thanks!